src/main/java/org/opensearch/security/filter/SecurityFilter.java

```diff
@@ -528,6 +529,10 @@ private boolean checkImmutableIndices(Object request, ActionListener listener) {
         return false;
     }
 
+    public void updatePluginToClusterAction(String pluginIdentifier, Set<String> clusterActions) {
+        evalp.updatePluginToClusterActions(pluginIdentifier, clusterActions);
```

Comment:
Nit: This method name uses `Action` (singular), while the called method name uses `Actions` (plural)

Reply:
Good catch. I pluralized the method name.